site/content/en/docs/Third Party Content/videos-and-presentations.md

```diff
@@ -1,12 +1,78 @@
 ---
 title: "Third Party Videos and Presentations"
-linkTitle: "Videos and Presentations"
+linkTitle: "Videos"
 date: 2019-01-03T08:37:35Z
 description: "Community contributed videos and presentations on Agones."
 weight: 10
 ---
 
-## Presentations
+## Case Studies
+
+### Engineering Without Limits: A Decade of Kubernetes Powering PlayStation’s Evolution (Kubecon NA, 2025)
+
+{{< youtube "goaBGk9wt0c" >}}
+
+### No More Disruption: PlayStation Network’s Approaches To Avoid Outages on Kubernetes Platform (Kubecon Japan, 2025)
+
+{{< youtube "xxSPRdwjuqE" >}}
+
+### (Paywall) 'Monster Hunter Wilds' Network Architecture Supporting Over 1 Million Concurrent Users (Presented by AWS) (Devcom, 2025)
+
+[![devcom-mhw-thumb.png](/images/devcom-mhw-thumb.png)](https://bizcommunity.gamescom.global/event/devcom-developer-conference-2025/planning/UGxhbm5pbmdfMjc0ODE4MQ==)
```

Comment:
Seems that it needs the relative path
```suggestion
[![devcom-mhw-thumb.png](../../../images/devcom-mhw-thumb.png)](https://bizcommunity.gamescom.global/event/devcom-developer-conference-2025/planning/UGxhbm5pbmdfMjc0ODE4MQ==)
```

Reply:
As you were mentioning on Slack about permission on this video, should we add a comment to let people know that they might not have access ?